packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -43,14 +44,13 @@ export async function verifyBlock(
 ): Promise<FullyVerifiedBlock> {
   const parentBlock = verifyBlockSanityChecks(chain, partiallyVerifiedBlock);
 
-  const {postState, executionStatus} = await verifyBlockStateTransition(chain, partiallyVerifiedBlock, opts);
+  const processedBlockData = await verifyBlockStateTransition(chain, partiallyVerifiedBlock, opts);
```

Comment:
Why this change? Being explicit and not doing the spread feels more readable in my opinion